features/step_definitions/login_steps.rb

```diff
@@ -0,0 +1,16 @@
+Given("I visit the {string} page") do |page_name|
+  visit page_path_from(page_name)
+end
+
+When("I click {string}") do |element|
+  click_link_or_button element
+end
+
+Then("I should be redirected to the {string} page") do |page_name|
+  expect(page.current_path).to eq page_path_from(page_name)
+end
+
+Then("I should be logged in as {string}") do |email|
+  user = User.find_by(email: email)
+  login_as(user, scope: :user)
```

Comment:
This is not an assertion. You are actually logging in that user with the `login_as(user, scope: :user)` so there's no way in knowing if your scenario has passed

Reply:
Okay that was my bad, will fix it asap :)

Reply:
Modified my step definition, not sure if it's exactly what you wanted, was the closest I could find to an assertion to check that the specific user has signed in.